doc/meson.build

```diff
@@ -36,31 +36,31 @@ if build_docbook
         install_dir: docs_dir,
       )
   endforeach
+endif
 
-  rst2man = find_program('rst2man', 'rst2man.py', required: false)
-  if rst2man.found()
-    rst2man_flags = [
-      '--syntax-highlight=none',
-    ]
-    
-    man_pages = [
-      { 'input': 'portals-conf.rst', 'output': 'portals.conf', 'section': '5' },
-    ]
+rst2man = find_program('rst2man', 'rst2man.py', required: get_option('man-pages'))
```

Comment:
Shouldn't need to do this twice.

Reply:
Sorry, I think I messed up with on my final revision.

Should be fixed now.